### ISSUE TITLE
Angular 8.2.0 + Storybook 5.2.0-beta.28 throws react warning

### ISSUE THREAD

**Issue by kroeder** [storybookjs/storybook#7746]
**Describe the bug**
Despite the known `:first-of-child` error from https://github.com/storybookjs/storybook/issues/6998 the latest `@next` throws 

```
vendors~main.8efd5a8e1001346d62f9.bundle.js:84111 Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: Draggable
```

**To Reproduce**
1. `npx @angular/cli new sb-example`
2. `npx @storybook/init@next init`
3. `npm run storybook`

**Expected behavior**
No warning

**Screenshots**
![image](https://user-images.githubusercontent.com/5478899/62897426-8d2d4400-bd53-11e9-9048-b458fc558d43.png)

**Comment by dontsave** [comment]
getting a componentwillreceiveprops warning in `5.2.0-rc.5` of @storybook/react as well

**Comment by imsunhao** [comment]
I also encountered the same problem using vue.

**Comment by slaviczavik** [comment]
Same issue for `@storybook/react` version `5.2.1`.

**Comment by frederickfogerty** [comment]
Getting the same for `@storybook/react` version `5.2.1`

**Comment by slaviczavik** [comment]
> Getting the same for `@storybook/react` version `5.2.1`

This issue was fixed in version **5.2.2**

**Comment by shilman** [comment]
Closing. Please upgrade to 5.2.2 and above!
